webapp/components/MapZoneSelector.tsx

```
@@ -0,0 +1,153 @@
+
+export const ZONE_NOZONE = 0;
+export const ZONE_METROPOLE = 1;
+export const ZONE_GUADELOUPE = 2;
+export const ZONE_MARTINIQUE = 3;
+export const ZONE_MAYOTTE = 4;
+export const ZONE_LAREUNION = 5;
+export const ZONE_GUYANE = 6;
+
+export default function MapZoneSelector(props: {
+  selectedZone: number;
+  zoneChangeCallback: (zone: number) => void | null;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)